user_tools/src/spark_rapids_tools/tools/qualx/preprocess.py

```diff
@@ -108,7 +108,11 @@
         'sqlOp_CustomShuffleReader',
         'sqlOp_DeserializeToObject',
         'sqlOp_Exchange',
-        'sqlOp_Execute InsertIntoHadoopFsRelationCommand',
+        'sqlOp_Execute InsertIntoHadoopFsRelationCommand csv',
+        'sqlOp_Execute InsertIntoHadoopFsRelationCommand parquet',
+        'sqlOp_Execute InsertIntoHadoopFsRelationCommand orc',
+        'sqlOp_Execute InsertIntoHadoopFsRelationCommand json',
+        'sqlOp_Execute InsertIntoHadoopFsRelationCommand unknown',
```

Comment:
missing `text`, and avro

Reply:
Thanks! Have added `text`. 
cc: @leewyang - Not sure if we need to add `avro` on both InsertIntoHadoopFsRelationCommand and Scan? Or if it is not required?

Reply:
If the ops from the Q tool will have `text` and `avro`, then we should probably add them.  That said, I don't think any of our current training datasets use text or avro, so these will basically be ignored even if we do add them.